crates/rnote-cli/src/export.rs

```diff
@@ -0,0 +1,673 @@
+    DocPages {
+        /// the directory the pages get exported to
+        #[arg(short = 'o', long)]
+        output_dir: PathBuf,
+        /// The directory the pages get exported to.{n}
+        /// Uses file stem of input file if not set.{n}
+        /// Cannot be used when exporting multiple files
+        #[arg(short = 's', long)]
+        output_file_stem: Option<String>,
+        /// the export output format
+        #[arg(short = 'f', long)]
+        output_format: DocPagesExportFormat,
+        /// The page order when documents with layouts that expand in horizontal and vertical directions are cut into pages
+        #[arg(long, default_value_t = Default::default())]
+        page_order: SplitOrder,
+        /// bitmap scale factor in relation to the actual size on the document
+        #[arg(long, default_value_t = DocPagesExportPrefs::default().bitmap_scalefactor)]
+        bitmap_scalefactor: f64,
+        /// quality of the jpeg image
+        #[arg(long, default_value_t = DocPagesExportPrefs::default().jpeg_quality)]
+        jpeg_quality: u8,
+    },
+    /// Export a selection in a document.{n}
+    /// When using --output-file, only one input file can be given.{n}
+    /// The export format is recognized from the file extension of the output file.{n}
+    /// When using --output-format, the same file name is used with the extension changed.{n}
+    /// --output-file and --output-format are mutually exclusive but one of them is required.{n}
+    /// Currently `.svg`, `.png` and `.jpeg` are supported.{n}
+    /// Available selection arguments - one of:{n}
+    /// --all: select all strokes{n}
+    /// --rect X Y width height: Select all strokes in given area{n}
+    /// When not using --all, use --bounds to switch between intersecting and inside bounds
+    #[command(alias = "sel")]
+    Selection {
+        #[command(flatten)]
+        file: FileArgs<SelectionExportFormat>,
+        #[command(subcommand)]
+        selection: SelectionCommands,
+        #[arg(short = 'c', long, default_value_t = Default::default(), global = true)]
+        /// if the strokes inside or intersecting with the given bounds are exported. Ignored when using all
+        selection_collision: SelectionCollision,
+        /// bitmap scale factor in relation to the actual size on the document
+        #[arg(long, default_value_t = SelectionExportPrefs::default().bitmap_scalefactor, global = true)]
+        bitmap_scalefactor: f64,
+        /// quality of the jpeg image
+        #[arg(long, default_value_t = SelectionExportPrefs::default().jpeg_quality, global = true)]
+        jpeg_quality: u8,
+        /// margin around the document
+        #[arg(long, default_value_t = SelectionExportPrefs::default().margin, global = true)]
+        margin: f64,
+    },
+}
+
+#[derive(Subcommand, Debug)]
+pub(crate) enum SelectionCommands {
+    /// Export all strokes
+    All,
+    /// Export a rectangular area of the document{n}
+    /// usage: X Y deltaX deltaY{n}
```

Comment:
remove this line, it is outdated.

Reply:
I moved the explanation of the inputs to the individual Arg documentations